### DISCUSSION TITLE
Future plas and roadmap for JSS

### DISCUSSION THREAD

**Discussion by Jony-Y** [cssinjs/jss#1620] [Q&A]
Hi,
We currently use React-JSS and are thinking about a revamp. 
Our original plan is to move to pure JSS and manage the sheets on our own. 
Looking into other frameworks I am trying to understand the roadmap for the JSS project. Future supports and perhaps added comparisons with SC, emotion and Linaria. 

For example performance, template literals support, and cost improvements of dynamic sheets.

Is there any document defining these items?

**Comment by Jony-Y** [comment]
Especially after the shift of MUI due to dynamic performance 

[See here](https://github.com/mui/material-ui/issues/22342)

**Reply by kof** [reply]
"The performance of our custom dynamic styles implementation" - mui had its own implementation for dynamic styles

**Reply by Jony-Y** [reply]
Is that to say that JSS does not have any performance issues with dynamic styles? Working according to the existing JSS docs and using dynamic styles will not cause any significant performance degradation?

**Comment by kof** [comment]
Generally I am currently not investing time in JSS, but I am happy  to merge good PRs
